internal/iapl/policy.go

```diff
@@ -208,6 +208,7 @@ func LoadPolicyDocumentFromFiles(filePaths ...string) (PolicyDocument, error) {
 }
 
 // LoadPolicyDocumentFromDirectory reads the provided directory path, reads all files in the directory, merges them, and returns a new merged PolicyDocument.
+// This does not recursively traverse directories.
```

Comment:
Recursive directories is something we want to keep to allow for better organization. It might be better to limit the directories traversed by filtering out directories prefixed with `.`.

Reply:
Sounds good. Let me update this PR accordingly.